tests/unit/js_utils/text_validation.spec.js

```diff
@@ -116,22 +116,23 @@ describe("TextValidation.validate_plate_barcode", () => {
     ["MA20044", "7"],
     ["MA20**#*", 8],
```

Comment:
As noted in earlier comment, include a description of what is right or wrong about the barcode in the array of test parameters

Reply:
As all of the test cases are less than 10 or 11 characters so now included in the title
`[input.length = %s]  < 10 or 11 and special characters are not matching criteria, Then validation fails and feedback text is <space>`  so has all details.